spec/lib/tasks/scripts/clean_bangladesh_phone_numbers_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require "tasks/scripts/clean_bangladesh_phone_numbers"
+
+RSpec.describe CleanBangladeshPhoneNumbers do
+  describe ".call" do
+    it "removes leading zeros from patient phone numbers" do
+      leading_zero_1 = create(:patient_phone_number, number: "01234567890")
+      leading_zero_2 = create(:patient_phone_number, number: "09876543210")
+
+      no_leading_zero_1 = create(:patient_phone_number, number: "1212121212")
+      no_leading_zero_2 = create(:patient_phone_number, number: "3434343434")
+
+      CleanBangladeshPhoneNumbers.call(verbose: false)
+
+      expect(leading_zero_1.reload.number).to eq("1234567890")
+      expect(leading_zero_2.reload.number).to eq("9876543210")
+
+      expect(no_leading_zero_1.reload.number).to eq("1212121212")
+      expect(no_leading_zero_2.reload.number).to eq("3434343434")
+    end
+
+    it "does nothing in a dryrun" do
+      leading_zero_1 = create(:patient_phone_number, number: "01234567890")
+      leading_zero_2 = create(:patient_phone_number, number: "09876543210")
+
+      no_leading_zero_1 = create(:patient_phone_number, number: "1212121212")
+      no_leading_zero_2 = create(:patient_phone_number, number: "3434343434")
```

Comment:
These could go in a `let`.

Reply:
They could 😄 I'm trying an Rspec approach that Rob showed me that minimizes `let` indirection by setting up test data inline. What do you think?

Reply:
The way I usually do these is to move up any obviously common data to the `lets` and setting up specific data inside the `its`. This _sort of_ seems to fit that bill.